src/Symfony/Component/Validator/Resources/translations/validators.de.xlf

```diff
@@ -330,6 +330,22 @@
                 <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
                 <target>Diese internationale Bankleitzahl (BIC) ist nicht mit der IBAN {{ iban }} assoziiert.</target>
             </trans-unit>
+            <trans-unit id="86">
```

Comment:
You updated the ids for the English translations but forgot to do the same here.

Reply:
done